frontend/src/components/PlannerCart/PlannerCart.jsx

```diff
@@ -48,7 +49,7 @@ const PlannerCart = () => {
         />
       </Tooltip>
       {openMenu && (
-        <div className="planner-cart-menu">
+        <S.plannerCartMenu style={{ backgroundColor: theme === "light" ? "#ffffff" : "#434343" }}>
```

Comment:
Could we use the `theme` prop within the styled component file instead? Rather than hardcoding the background color here?